core/bin/dust_api.rs

```diff
@@ -1745,187 +1753,25 @@ async fn data_sources_delete(
     }
 }
 
-// Databases
-
-#[derive(serde::Deserialize)]
-struct DatabasesRegisterPayload {
-    database_id: String,
-    name: String,
-}
-
-async fn databases_register(
```

Comment:
all `databases` related endpoints are deleted except for `/query_database`

Reply:
:+1: